lib/scanner/index.js

```diff
@@ -156,7 +157,13 @@ class Scanner {
         this.callback(data);
       })
 
-      .catch(err => console.error(err));
+      .catch(err => {
+        console.info(err);
+        if (['NetworkError', 'TypeError'].indexOf(err.name) 
```

Comment:
We are also receiving a TypeError: Network error message in some cases, so I added both to the list of error problems.